ui/app/configuration/configuration-controllers.js

```diff
@@ -27,25 +27,21 @@
     'configurationList',
     'PncRestClient',
     function($log, $state, configurationList, PncRestClient) {
-      var self = this;
-      $log.debug('ConfigurationListController >> this=%O, configurationList=%O',
-                 self, configurationList);
-
-      self.configurations = configurationList;
-      self.projects = [];
-
-      angular.forEach(self.configurations, function(configuration){
-
-          PncRestClient.Project.get({
-              projectId: configuration.projectId
-          }).$promise.then(
-            function (result) {
-              if (result) {
-                self.projects.push(result);
-                //console.log(JSON.stringify(self.projects));
-              }
+      var that = this;
+
+      this.configurations = configurationList;
+      this.projects = [];
+
+      angular.forEach(this.configurations, function(configuration) {
```

Comment:
To be consistent with other controllers, let's put:

var self = this;

self.configurations = configurationList;
self.projects = [];

angular.forEach(self.configurations, function(configuration) {